keras/models.py

```diff
@@ -979,7 +979,7 @@ def fit_generator(self, generator, samples_per_epoch, nb_epoch,
                       verbose=1, show_accuracy=False, callbacks=[],
                       validation_data=None, nb_val_samples=None,
                       class_weight=None,
-                      nb_worker=1, nb_val_worker=None):
+                      nb_worker=1, nb_val_worker=None, batch_size=128):
```

Comment:
Please add this to the docstring, with mention that it only applies to the validation data.